docs/code/IDataViewImplementation.md

```diff
@@ -337,13 +337,13 @@ aside (which we can hardly help), we expect the models to be the same.
 When you create a loader you have the option of specifying not only *one* data
 input, but any number of data input files, including zero. But there's also a
 more general principle at work here with zero files: when deserializing a data
-loader from a data model with an `IMultiStreamSource` with `Count == 0` (e.g.,
+loader from a data model with an `IMultiStreamSource` with `Count == 0` (for example,
```

Comment:
It's a little unfortunate that the linebreaking got all screwed up due to the longer lines, but I guess maybe we can opportunistically fix it later?

Reply:
@TomFinley this is usually a personal choice. Some people choose to cut the text that way, no?